apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -649,6 +651,14 @@ export class RushConfiguration {
     return this._gitSampleEmail;
   }
 
+  /**
+   * [Part of the "gitPolicy" feature.]
+   * A default commit message to use when committing chnages during the Rush lifecycle.
```

Comment:
I would change this to "The commit message to use when committing changes during 'rush publish'"